### PR TITLE
test_spec: Add CI-secdom-test

### PR DESCRIPTION
**Pull request by eirikekbjoe**
Adding new test specification for public secdom test plan.

Jira: NCSDK-28744

### CODE REVIEW

**Review by harish1992 — APPROVED** [review]
looks good

File: test-manifests/99-default-test-nrf.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ manifest:
     - name: test_nrf
       remote: ncs-test
       repo-path: test-fw-nrfconnect-nrf
-      revision: master
+      revision: feature/secdom-samples-public
```
Comment (jangalda-nsc) [inline test-manifests/99-default-test-nrf.yml:14]: Please revert

**Review by jangalda-nsc — CHANGES_REQUESTED** [review]

File: .github/test-spec.yml [hunk 2]
```diff
@@ -599,3 +599,20 @@
   - "samples/nrf_rpc/protocols_serialization/**/*"
   - "subsys/net/openthread/rpc/**/*"
   - "subsys/nrf_rpc/**/*"
+
+"CI-secdom-test":
+  - "samples/bluetooth/multiple_adv_sets/**/*"
+  - "samples/nfc/record_text/**/*"
```
Comment (miha-nordic) [inline .github/test-spec.yml:605]: Why SecDom CI should be run on NFC and applications changes?

Comment (eirikekbjoe) [reply .github/test-spec.yml:605]: We are on a later revision of sdk-nrf and want to test our newest release with samples built from latest sdk-nrf.

**Review by jangalda-nsc — APPROVED** [review]

File: .github/test-spec.yml (hunk 2 above)
Comment (ChristopherBittner) [reply .github/test-spec.yml:605]: > Why SecDom CI should be run on NFC and applications changes?

That sample is used to test peripheral access configuration via secdom.

**Review by ChristopherBittner — APPROVED** [review]

**Review by miha-nordic — APPROVED** [review]

**Pull request merged**